app/Http/Controllers/CompetitionsController.php

```diff
@@ -16,7 +25,8 @@ class CompetitionsController extends Controller
      */
     public function index()
     {
-        return view('competitions.index');
+        $competitions = Competition::all();
+        return view('competitions.index')->withCompetitions($competitions);
```

Comment:
`withCompetitions($competitions);` that seems silly and repetitive..is that real?!